gamemodes/terrortown/gamemode/client/cl_eventpopup.lua

```diff
@@ -61,8 +62,8 @@ function EPOP:AddMessage(title, subtitle, displayTime, iconTable, blocking)
 	-- add the new message to the queue
 	self.messageQueue[queueSize + 1] = {
 		id = id,
-		title = isstring(title) and {text = title} or title,
-		subtitle = isstring(subtitle) and {text = subtitle} or subtitle,
+		title = (not title or isstring(title)) and {text = title or ""} or title,
```

Comment:
why not using `title = title or ""` in front to avoid the first check and the second `or` that does somehow the same?

Reply:
because it has to be in a table if it is not already a table

Reply:
That answer doesn't match my question

Reply:
an aternative would be
```lua
title = title or ""
title = istable(title) and title or {text = title}
```
and I don't see any benefit here

Reply:
```lua
title = title or ""
subtitle = subtitle or ""

-- add the new message to the queue
self.messageQueue[queueSize + 1] = {
	id = id,
	title = isstring(title) and {text = title} or title,
	subtitle = isstring(subtitle) and {text = subtitle} or subtitle,
	displayTime = displayTime or 4,
-- ...
```

Reply:
this is exactly what I've just written in the previous comment and I don't see any benefit here except adding another line. Especially since title can be a table, the `string|table or ""` is rather confusing

Reply:
and `title = (not title or isstring(title)) and {text = title or ""} or title,` is better? :D

Reply:
In my opinion, yes. It is a single line. And it says clearly that the title is converted to a table if is undefined or a string and kept if is a table

Reply:
Okay, it's a very small change, but here we go: Your solution needs more checks:
1. Is a title given? No.
2. Set text ? title. But title is nil. So set it "".

Mine:
1. Is a title given? No. Set it to "".

Additionally, while looking through the code, my solution makes clear it's a string if it's nil. And is showing that it's creating a table if it's a string.

It isn't every time the best solution to keep things compressed. In code, we can try to make it clear. Furthermore, the doc on top is telling the Dev, that a string or a table is useable, so that's an invalid arg imo.

Reply:
> 1. Is a title given? No. Set it to "".

and 2. Is it a table? No. Set it to `{text = title}`